## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 435368`
Commit: `673da0e7855672d81645b9f025be222b225a7a9c`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>ansible-later</li>
  </ul>
</details>
<details>
  <summary>:x: 14 packages failed to build:</summary>
  <ul>
    <li>ansible-navigator</li>
    <li>ansible-navigator.dist</li>
    <li>python312Packages.ansible-kernel</li>
    <li>python312Packages.ansible-kernel.dist</li>
    <li>python312Packages.ansible-runner</li>
    <li>python312Packages.ansible-runner.dist</li>
    <li>python312Packages.pytest-ansible</li>
    <li>python312Packages.pytest-ansible.dist</li>
    <li>python313Packages.ansible-kernel</li>
    <li>python313Packages.ansible-kernel.dist</li>
    <li>python313Packages.ansible-runner</li>
    <li>python313Packages.ansible-runner.dist</li>
    <li>python313Packages.pytest-ansible</li>
    <li>python313Packages.pytest-ansible.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 28 packages built:</summary>
  <ul>
    <li>ansible (ansible_2_18, python313Packages.ansible-core)</li>
    <li>ansible-doctor</li>
    <li>ansible-doctor.dist</li>
    <li>ansible-lint</li>
    <li>ansible-lint.dist</li>
    <li>ansible.dist (ansible_2_18.dist, python313Packages.ansible-core.dist)</li>
    <li>molecule (python313Packages.molecule)</li>
    <li>molecule.dist (python313Packages.molecule.dist)</li>
    <li>python312Packages.ansible-compat</li>
    <li>python312Packages.ansible-compat.dist</li>
    <li>python312Packages.ansible-core</li>
    <li>python312Packages.ansible-core.dist</li>
    <li>python312Packages.ansible-vault-rw</li>
    <li>python312Packages.ansible-vault-rw.dist</li>
    <li>python312Packages.dynaconf</li>
    <li>python312Packages.dynaconf.dist</li>
    <li>python312Packages.molecule</li>
    <li>python312Packages.molecule.dist</li>
    <li>python312Packages.pytest-testinfra</li>
    <li>python312Packages.pytest-testinfra.dist</li>
    <li>python313Packages.ansible-compat</li>
    <li>python313Packages.ansible-compat.dist</li>
    <li>python313Packages.ansible-vault-rw</li>
    <li>python313Packages.ansible-vault-rw.dist</li>
    <li>python313Packages.dynaconf</li>
    <li>python313Packages.dynaconf.dist</li>
    <li>python313Packages.pytest-testinfra</li>
    <li>python313Packages.pytest-testinfra.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python312Packages.ansible-runner</summary>
<pre>test/integration/test_transmit_worker_process.py: 9 warnings
  /nix/store/18l19gnl8dq75v6knghbcgkrikmpc0yv-python3-3.12.11/lib/python3.12/pty.py:95: DeprecationWarning: This process (pid=309) is multi-threaded, use of forkpty() may lead to deadlocks in the child.
    pid, fd = os.forkpty()

test/integration/test___main__.py: 2 warnings
test/integration/test_config.py: 3 warnings
test/integration/test_display_callback.py: 16 warnings
  /nix/store/18l19gnl8dq75v6knghbcgkrikmpc0yv-python3-3.12.11/lib/python3.12/pty.py:95: DeprecationWarning: This process (pid=303) is multi-threaded, use of forkpty() may lead to deadlocks in the child.
    pid, fd = os.forkpty()

test/integration/test_main.py::test_role_start
test/integration/test_main.py::test_playbook_start
  /nix/store/18l19gnl8dq75v6knghbcgkrikmpc0yv-python3-3.12.11/lib/python3.12/multiprocessing/popen_fork.py:66: DeprecationWarning: This process (pid=309) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED test/integration/test_display_callback.py::test_output_when_given_invalid_playbook - AssertionError: assert 'ERROR! the playbook:' in '\x1b[0;31m[ERROR]: the pl...
FAILED test/integration/test_events.py::test_include_role_events - AssertionError: assert None == 'ansible.builtin.debug'
========== 2 failed, 2033 passed, 56 skipped, 262 warnings in 57.11s ===========</pre>
</details>
<details>
<summary>python312Packages.pytest-ansible</summary>
<pre>adding 'pytest_ansible/host_manager/v213.py'
adding 'pytest_ansible/module_dispatcher/__init__.py'
adding 'pytest_ansible/module_dispatcher/v213.py'
adding 'pytest_ansible-25.6.3.dist-info/licenses/LICENSE'
adding 'pytest_ansible-25.6.3.dist-info/METADATA'
adding 'pytest_ansible-25.6.3.dist-info/WHEEL'
adding 'pytest_ansible-25.6.3.dist-info/entry_points.txt'
adding 'pytest_ansible-25.6.3.dist-info/top_level.txt'
adding 'pytest_ansible-25.6.3.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
Successfully built pytest_ansible-25.6.3-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for pytest_ansible-25.6.3-py3-none-any.whl
  - pytest-plus not installed
  - pytest-sugar not installed
  - pytest-xdist not installed</pre>
</details>
<details>
<summary>python313Packages.ansible-runner</summary>
<pre>test/integration/test_transmit_worker_process.py: 9 warnings
  /nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/pty.py:95: DeprecationWarning: This process (pid=309) is multi-threaded, use of forkpty() may lead to deadlocks in the child.
    pid, fd = os.forkpty()

test/integration/test___main__.py: 2 warnings
test/integration/test_config.py: 3 warnings
test/integration/test_display_callback.py: 16 warnings
  /nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/pty.py:95: DeprecationWarning: This process (pid=303) is multi-threaded, use of forkpty() may lead to deadlocks in the child.
    pid, fd = os.forkpty()

test/integration/test_main.py::test_role_start
test/integration/test_main.py::test_playbook_start
  /nix/store/iyff8129iampdw13nlfqalzhxy8y1hi9-python3-3.13.6/lib/python3.13/multiprocessing/popen_fork.py:67: DeprecationWarning: This process (pid=309) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED test/integration/test_display_callback.py::test_output_when_given_invalid_playbook - AssertionError: assert 'ERROR! the playbook:' in '\x1b[0;31m[ERROR]: the pl...
FAILED test/integration/test_events.py::test_include_role_events - AssertionError: assert None == 'ansible.builtin.debug'
========== 2 failed, 2033 passed, 56 skipped, 263 warnings in 53.81s ===========</pre>
</details>
</details>

--- a/pkgs/development/python-modules/ansible/core.nix
+++ b/pkgs/development/python-modules/ansible/core.nix
@@ -32,15 +32,15 @@
 
 buildPythonPackage rec {
   pname = "ansible-core";
-  version = "2.18.7";
+  version = "2.19.1";
   pyproject = true;
 
-  disabled = pythonOlder "3.11";
+  disabled = pythonOlder "3.12";
 
   src = fetchPypi {
     pname = "ansible_core";
     inherit version;
-    hash = "sha256-GhKb+fzV3KKxfoPOdxR+4vvDxRpJWJcBUol8xbbQquc=";
+    hash = "sha256-r/0zs40ytXz8LNba86r8s4QpcDnkxWABlqLumqAnt10=";
   };
 
   # ansible_connection is already wrapped, so don't pass it through

